Submitty/Submitty
# [Refactor:Router] New URL for SubmissionController

### URL Changes

- GET `http://192.168.56.111/f19/sample/gradeable/<gradeable_id>`
- GET `http://192.168.56.111/f19/sample/gradeable/<gradeable_id>/<gradeable_version>`
- POST `http://192.168.56.111/f19/sample/gradeable/<gradeable_id>/version/<new_version>` (update version for grading)
- POST `http://192.168.56.111/f19/sample/gradeable/<gradeable_id>/upload`
- POST `http://192.168.56.111/f19/sample/gradeable/<gradeable_id>/verify`
- POST `http://192.168.56.111/f19/sample/gradeable/<gradeable_id>/bulk`
- POST `http://192.168.56.111/f19/sample/gradeable/<gradeable_id>/split_pdf/upload`
- POST `http://192.168.56.111/f19/sample/gradeable/<gradeable_id>/split_pdf/delete`
- GET `http://192.168.56.111/f19/sample/gradeable/<gradeable_id>/<gradeable_version>/check_refresh`
- GET `http://192.168.56.111/f19/sample/gradeable/<gradeable_id>/bulk_stats`

### Testing
Manually tested the following:
- Simple upload
- Bulk upload
- Assign split pdfs to students
- Delete split pdfs
- See bulk stats
- Update versions

SubmissionController is an important part of Submitty, without which we will only get `ty`, so I did not modify any inner logic. However, I am not sure about some parts of submission, like `merge` and `clobber`. Would appreciate it if someone help review that.
